Package.swift

```diff
@@ -18,7 +18,8 @@ let package = Package(
         .watchOS(.v2),
     ],
     products: [
-        .library(name: "Conduit", targets: ["Conduit"]),
+        .library(name: "ConduitStatic", targets: ["Conduit"]),
+        .library(name: "Conduit", type: .dynamic, targets: ["Conduit"]),
```

Comment:
This will change the current target from static to dynamic, correct? Would it be safer to instead create a `ConduitDynamic` target?

Reply:
It seemed to me that for most of our apps we were going to need the dynamic one, so I made that one the "default". But yes, we can do it that way. It would prevent any impact on anyone currently using `Conduit` via SwiftPM without this issue.

Will switch them now.